manifests/params.pp

```diff
@@ -159,7 +159,12 @@
       $perl_package_name      = pick($perl_package_name, 'libdbd-pg-perl')
       $plperl_package_name    = pick($plperl_package_name, "postgresql-plperl-${version}")
       $plpython_package_name  = pick($plpython_package_name, "postgresql-plpython-${version}")
-      $python_package_name    = pick($python_package_name, 'python-psycopg2')
+
+      if $facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['full'], '22.04') >= 0 {
```

Comment:
Looks like the same is needed at least for Debian 12. For Debian 11 there still is `python-psycopg2` as well as `python3-psycopg2`. Would you mind doing the same for Debian >= 12 at least while you're at it?

Reply:
I have added such a test for Debian 12 as well, trying to make the logic somewhat readable. Not sure if this is in tune with the module's coding style though.